src/lib/components/Footers.svelte

```diff
@@ -1,8 +1,8 @@
 <script>
-    import CFooter from "$lib/components/CFooter.svelte";
-    import SFooter from "$lib/components/SFooter.svelte"
+    import CFooter from '$lib/components/CFooter.svelte';
+    import SFooter from '$lib/components/SFooter.svelte';
```

Comment:
Just a nitpick, but it would be great if we could use a more descriptive import name and file name for the footer. 👌